workbench/context/ConnectionConfiguration.php

```diff
@@ -47,7 +53,7 @@ function applyLoginResult($loginResult) {
         $this->sessionId = crypto_serialize($loginResult->sessionId);
     }
 
-    static function fromUrl($serviceUrl, $sessionId, $clientId) {
+    static function fromUrl($serviceUrl, $sessionId, $clientId, $ignoreHTTPSCertificateIssues=false) {
```

Comment:
don't know enough about PHP to judge whether this is a good idea or I need to add parameter everywhere explicitly